libffi-sys-rs/Cargo.toml

```diff
@@ -20,3 +20,6 @@ features = ["system"]
 
 [target.'cfg(target_env = "msvc")'.build-dependencies]
 cc = "1.0.48"
+
+[build-dependencies]
+cc = "1.0.73"
```

Comment:
1. Patch versions aren't needed, instead `cc = "^1.0"` should be good enough
2. If we always require `cc` as a build dependency, we don't need it as a conditional dependency above